app/mixins/event-wizard.js

```diff
@@ -59,7 +59,14 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
       }
     }
     const numberOfTickets = data.tickets ? data.tickets.length : 0;
-    if (event.name && event.locationName && event.startsAtDate && event.endsAtDate && numberOfTickets > 0) {
+    let minMaxValueOfTicketValid = true;
+    for (const ticket of data.tickets ? data.tickets.toArray() : []) {
+      if (!(parseInt(ticket.get('maxOrder')) >= parseInt(ticket.get('minOrder')))) {
+        minMaxValueOfTicketValid = false;
+        break;
+      }
+    }
+    if (event.name && event.locationName && event.startsAtDate && event.endsAtDate && numberOfTickets > 0 && minMaxValueOfTicketValid) {
```

Comment:
Not required in this way , as server will throw error error too for the same. If you can go in way like Semsntic UI validations, then the PR would make sense, as I will have to click submit button to fire up this check

Reply:
> I think the best place to add the check will be where we are getting data to create the event. Add a check just before hitting server. There are many checks implemented similarly.
Validation rules will not be helpful as you are using two different entities here.

It's mentioned in the issue

Reply:
Also, why required to hit the server, when the client side data is provided wrong? It makes sense that always the ticketMaxOrder will be greater than equal to ticketMinOrder We can add the check frontend side

Reply:
@kushthedude